### PR TITLE
Add Periods to Short, Long Dec and Inline Comments

### PR DESCRIPTION
**Pull request by ahmadawais**
This commit Adda Periods to Short, Long Dec and Inline Comments in accordance with WP JS documentation standards.

### CODE REVIEW

File: wp-api.js [hunk 1]
```diff
@@ -10,8 +10,8 @@
 	};
 
 	/**
-	 * ECMAScript 5 shim, from MDN
-	 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
+	 * ECMAScript 5 shim, from MDN.
+	 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString.
```
Comment (danielbachhuber) [inline wp-api.js]: I'm not sure it makes sense to add a period here, as it changes the nature of the URL.

File: wp-api.js [hunk 2]
```diff
@@ -278,9 +278,9 @@
 			idAttribute: 'id',
 
 			/**
-			 * Return URL for the model
+			 * Return URL for the model.
 			 *
-			 * @returns {string}
+			 * @returns {string}.
```
Comment (danielbachhuber) [inline wp-api.js]: Same comment as https://github.com/WP-API/WP-API/pull/1702/files#r43246164

File: wp-api.js (hunk 1 above)
Comment (ahmadawais) [inline wp-api.js]: Let me re-submit this PR and I will add `@link` tags for these URLs without a period at the end, as suggested in the core handbook.

File: wp-api.js (hunk 2 above)
Comment (ahmadawais) [inline wp-api.js]: I have answered that comment on the other PR.

File: wp-api.js [hunk 3]
```diff
@@ -79,27 +79,27 @@
...
-	 * @type {{toJSON: toJSON, parse: parse}}
+	 * @type {{toJSON: toJSON, parse: parse}}.
 	 */
 	var TimeStampedMixin = {
 		/**
-		 * Serialize the entity pre-sync
+		 * Serialize the entity pre-sync.
 		 *
-		 * @returns {*}
+		 * @returns {*}.
```
Comment (danielbachhuber) [inline wp-api.js]: No period

File: wp-api.js [hunk 4]
```diff
@@ -110,13 +110,13 @@
 		},
 
 		/**
-		 * Unserialize the fetched response
+		 * Unserialize the fetched response.
 		 *
-		 * @param {*} response
-		 * @returns {*}
+		 * @param {*} response.
+		 * @returns {*}.
```
Comment (danielbachhuber) [inline wp-api.js]: No period

**Comment by danielbachhuber** [comment]
There are a number of instances where the period needs to be removed.

**Comment by ahmadawais** [comment]
@danielbachhuber I have updated the file and removed the unnecessary periods.

**Pull request merged**